gen-protoc.sh

```diff
@@ -4,4 +4,4 @@
 protoc --python_out=aioesphomeapi -I aioesphomeapi aioesphomeapi/*.proto
 
 # https://github.com/protocolbuffers/protobuf/issues/1491
-sed -i '' 's/import api_options_pb2 as api__options__pb2/from . import api_options_pb2 as api__options__pb2/' aioesphomeapi/api_pb2.py
+sed -i 's/import api_options_pb2 as api__options__pb2/from . import api_options_pb2 as api__options__pb2/' aioesphomeapi/api_pb2.py
```

Comment:
Ugh, this is an issue with different sed implementations - on my macOS system the extra '' is needed. I guess I will just alias sed to gsed to fix it here.